src/sentry/api/bases/organization_events.py

```diff
@@ -325,29 +326,52 @@ def get_event_stats_data(
                 for key, event_result in result.items():
                     if len(query_columns) > 1:
                         results[key] = self.serialize_multiple_axis(
-                            serializer, event_result, columns, query_columns, allow_partial_buckets
+                            serializer,
+                            event_result,
+                            columns,
+                            query_columns,
+                            allow_partial_buckets,
+                            zerofill_results=zerofill_results,
                         )
                     else:
                         # Need to get function alias if count is a field, but not the axis
                         results[key] = serializer.serialize(
                             event_result,
                             column=resolve_axis_column(query_columns[0]),
                             allow_partial_buckets=allow_partial_buckets,
+                            zerofill_results=zerofill_results,
                         )
-                return results
+                serializedResult = results
             elif len(query_columns) > 1:
-                return self.serialize_multiple_axis(
-                    serializer, result, columns, query_columns, allow_partial_buckets
+                serializedResult = self.serialize_multiple_axis(
+                    serializer,
+                    result,
+                    columns,
+                    query_columns,
+                    allow_partial_buckets,
+                    zerofill_results=zerofill_results,
                 )
             else:
-                return serializer.serialize(
+                serializedResult = serializer.serialize(
                     result,
                     resolve_axis_column(query_columns[0]),
                     allow_partial_buckets=allow_partial_buckets,
+                    zerofill_results=zerofill_results,
                 )
+            if hasattr(result, "start") and hasattr(result, "end"):
+                serializedResult["start"] = result.start
+                serializedResult["end"] = result.end
```

Comment:
From slack, we cant add `start` and `end` as top-level keys when we have a multi-axis graph as the top-level keys are treated as series names.

Reply:
Good feedback! Updated to add `start` and `end` to within the series object.